cscs-checks/prgenv/helloworld.py

```diff
@@ -109,7 +109,8 @@ def compile_timer_end(self):
 class HelloWorldTestSerial(HelloWorldBaseTest):
     def __init__(self, lang, linkage):
         super().__init__('serial', lang, linkage)
-        self.valid_systems += ['arolla:pn', 'tsa:pn']
```

Comment:
Why did you remove this?

Reply:
Because these systems are already in the valid systems of `HelloWorldBaseTest`.